fix(vmclass): use qemu64 CPU model for Discovery and Features types to improve AMD compatibility (#1446)
deckhouse/virtualization: docs/internal/cpu_model_type_discovery.md

The error might be a bug in libvirt when it compares features after resolving the target CPU model (still
need to investigate).

The current approach is to use qemu64 model for Discovery and Features types. This model contains a small

> set of features and migration works well. Changed from kvm64 to qemu64 to improve compatibility with AMD CPUs.

Reviewer: ```suggestion
set of features and migration works well. (We choose qemu64 over the kvm64 because of better compatibility with AMD CPUs).
```
